internal/common/README.md

```diff
@@ -36,4 +36,4 @@ To enforce an update on those files, you can delete them and run the code again/
 
 ### Usage:
 
-The package is designed to be used by calling the `SetMockAVSs()` function, which sets up the data structures with the latest versions of the repositories. The data is then written to the `mock-avs-versions.yml` file. If the data is older than an hour, it fetches the latest data from GitHub and updates the file.
\ No newline at end of file
+The package is designed to be used by calling the `SetMockAVSs()` function, which sets up the data structures with the latest versions of the repositories. The data is then written to the `mock-avs-versions.yml` file. If the data is older than an hour, it fetches the latest data from GitHub and updates the file.
```

Comment:
I don't see any difference here. not sure why github added this. can you revert somehow ? @omahs

Reply:
Hey sure, just fixed this